src/main/java/edu/ucdavis/dss/ipa/entities/InstructorTypeCost.java

```diff
@@ -0,0 +1,122 @@
+package edu.ucdavis.dss.ipa.entities;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.persistence.Transient;
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import edu.ucdavis.dss.ipa.api.deserializers.InstructorTypeCostDeserializer;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@SuppressWarnings("serial")
+@Entity
+@Table(name = "InstructorTypeCosts")
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+@JsonDeserialize(using = InstructorTypeCostDeserializer.class)
+public class InstructorTypeCost extends BaseEntity {
+    private long id;
+    private Budget budget;
+    private Float cost;
+    private InstructorType instructorType;
+    private List<InstructorCost> instructorCosts = new ArrayList<>();
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "Id", unique = true, nullable = false)
+    @JsonProperty
+    public long getId()
+    {
```

Comment:
Opening bracket belongs on previous line. Please follow existing style.

Reply:
Bah, code folding hid that from me with single line function collapsing. Dug into the settings and disabled it.